feder/main/templates/base.html

```diff
@@ -71,25 +71,26 @@ <h1 class="sr-only">{% trans 'Civil data mining' %}</h1>
                     </a>
                     <ul class="dropdown-menu">
                         <li>
+                            {# performant paginator does not support last #}
                             <a href="{% url 'cases:list' %}">
                                 <i class="fa fa-folder-open"></i>
                                 {% trans 'Cases' %}
                             </a>
                         </li>
                         <li>
-                            <a href="{% url 'monitorings:list' %}">
+                            <a href="{% url 'monitorings:list' %}?page=last">
```

Comment:
I don't like change order of index due jump of element over pages, but I see value to link latest page by default.